cron_tasks_folder/daily1pm/send-mail

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+echo "Sending mail $(date)"
+ssmtp xpub-tech-alerts@elifesciences.org < /mail-config/test.msg
```

Comment:
I assume sending a fixed email every day is not the intended behavior; can a realistic example to be provided to check the integration of what is running in this script with the rest of the application around it?

Reply:
Yup - this is my lean approach... I would like to deploy this with the fixed email initiailly though before the scripts are added :)

Reply:
Can you paste in a comment some sample code that will end up here though?

Reply:
Done